--- a/core/bug_api.php
+++ b/core/bug_api.php
@@ -1910,6 +1910,12 @@ function bug_get_attachments( $p_bug_id ) {
 	global $g_cache_bug_attachments;
 	if( isset( $g_cache_bug_attachments[$p_bug_id] ) ) {
 		return $g_cache_bug_attachments[$p_bug_id];
 	}
 
+	# VEditor begin
+	if( function_exists( 'veditor_bug_get_attachments' ) ) {
+		return veditor_bug_get_attachments( $p_bug_id );
+	}
+	# VEditor end
+
 	db_param_push();
